PRIME-514 - Angular 9

mtpultz opened the pull request.

hogmann commented on prime-angular-frontend/Dockerfile:
```diff
@@ -19,37 +19,36 @@ ENV KEYCLOAK_CLIENT_ID $KEYCLOAK_CLIENT_ID
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix && \
```
Not sure if this is needed commented out here as we are including it above.

mtpultz replied:
Yah, probably not, but it's Paul's code and I don't know if he had something in mind when he commented it out.  @poggdogg do we need this commented out code in the Dockerfile?

poggdogg replied:
Was commented out during troubleshooting of angular 8 - please delete.

hogmann commented on prime-angular-frontend/openshift.dockerfile:
```diff
@@ -36,15 +36,14 @@ COPY nginx.template.conf /etc/nginx/nginx.template.conf
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix --only=prod && \
```
Same here

mtpultz replied:
@poggdogg do we need this commented out code in the Dockerfile?

mtpultz commented:
@poggdogg I've updated the Dockerfile and OpenShift template.  I've removed our temporary patch and turned on audit fix in the other environments since I'd rather find out there's an issue with it updating something before we push to prod, or even test.
